WISE-Community/WISE
# Issue 1428 do not require google client id

This changes 2 things:
1. If you don't specify google oauth id's in wise.properties, WISE should still start up, and sign in/register with google buttons should be hidden from view.
2. The CI tests pass again (there were more to making the tests pass than fixing #1428, but I worked on it also on this branch).

How to verify:
1. Comment out lines in wise.properties that are related to google authentication, and start WISE. It should start without issues after pulling the changes.
2. Check that Sign In/Register with Google buttons are no longer visible for students and teachers.
3. Check that CI tests are passing again in Travis CI.


I also realized that we don't need to put the stub service definitions inside beforeEach in the spec files. This may reduce test run times...I will experiment with this in another branch. 

#1428
